### PR TITLE
Issue/dummy plevels

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

- create new NationalForecastValue pydantic model
- create dummy porpties if 

Fixes #

## How Has This Been Tested?

- CI tests
- added new test to check dummy default values are made

- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/national.py [hunk 1]
```diff
@@ -57,15 +68,43 @@ def get_national_forecast(
...
+            logger.warning(
+                f"Using default properties for {national_forecast_value.__fields__.keys()}"
+            )
+            national_forecast_value.plevels = {
+                "plevel_10": national_forecast_value.expected_power_generation_megawatts * 0.8,
+                "plevel_90": national_forecast_value.expected_power_generation_megawatts * 1.2,
+            }
+            logger.debug(f"{national_forecast_value.plevels}")
+
+        for c in ["10", "90"]:
```
Comment (peterdudfield) [inline src/national.py:101]: add comment renaming

File: src/national.py [hunk 2]
```diff
@@ -57,15 +68,44 @@ def get_national_forecast(
...
+    # change to NationalForecastValue
+    national_forecast_values = []
+    for f in forecast_values:
+        # change to NationalForecastValue
+        plevels = f._properties
+        national_forecast_value = NationalForecastValue(**f.__dict__)
+        national_forecast_value.plevels = plevels
+
+        # add default values in, we will remove this at some point
+        if not isinstance(national_forecast_value.plevels, dict):
```
Comment (devsjc) [inline src/national.py:91]: Does this trigger on an empty dict as well as things that aren't dicts?

**Review by devsjc — APPROVED** [review]

File: src/national.py (hunk 2 above)
Comment (peterdudfield) [reply src/national.py:91]: hmm, thats a good one, I should add that case

File: src/tests/test_national.py [hunk 3]
```diff
@@ -1,26 +1,63 @@
...
+    national_forecast_values = [NationalForecastValue(**f) for f in response.json()]
+    assert national_forecast_values[0].plevels is not None
+    # index 24 is the middle of the day
+    assert (
+        national_forecast_values[24].plevels["plevel_10"]
+        != national_forecast_values[0].expected_power_generation_megawatts * 0.9
+    )
+
+
+def test_read_latest_national_values_no_properotes(db_session, api_client):
```
Comment (rachel-labri-tipton) [inline src/tests/test_national.py:45]: tiny detail, but should the test be `test_read_latest_national_values_no_properties`?

**Review by rachel-labri-tipton — APPROVED** [review]

File: src/tests/test_national.py (hunk 3 above)
Comment (peterdudfield) [reply src/tests/test_national.py:45]: yea, thanks for the typo

**Pull request merged**